src/web/modules/custom/workbc_cdq_career_match/src/Plugin/Block/ReturnToResultsBlock.php

```diff
@@ -38,11 +33,12 @@ public function build() {
       'query' => ['token' => $submission->getToken()]
     ])->toString();
 
-    $markup = "";
-    $markup .= '<div class="cdq-back-link">';
-    $markup .= '<a href="#back" class="back-to-results"><img src="/themes/custom/workbc_cdq/assets/arrow-left.svg"/> Back to Quiz Results </a>';
-    $markup .= '</div>';
-  
+    $markup = <<<END
+      <div class="cdq-back-link">
+        <a href="$results_link" class="back-to-results"><img src="/themes/custom/workbc_cdq/assets/arrow-left.svg"/>Back to Quiz Results</a>
```

Comment:
This is the crux of the fix. @rjkuyvenhoven , any specific reason why `$results_link` was not used previously?

Reply:
Yes, because using $results_link resulted in sorting not being retained when returning to the results page.

see WBCAMS-616

Reply:
ok thanks, I'll make sure both work